src/auth/auth-router.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const manager = require('./auth-manager');
+const router = express.Router();
```

Comment:
Partially to make it abundantly clear that the manager is the auth manager (rather than some other manager that a reader might think it is), and partially for consistency with events/users/reports.

A reader could probably deduce fairly quickly that the manager is the auth manager, since it's in the auth domain, but I think having ```auth``` as a prefix would still make it a little clearer.